src/services/TaskService.ts

```diff
@@ -62,11 +64,12 @@ export default class TaskService {
   }
 
   //Create a task
-  createTask(projectId: string, title: string, description: string): Promise<Task> {
+  createTask(projectId: string, title: string, description: string, ownerId?: string): Promise<Task> {
     const newTask: Partial<Task> = {
       title: title,
       description: description,
       status: Status.OPEN,
+      ownerId: ownerId,
```

Comment:
The issue with the owner and ownerId has already been discussed in the comments of my pull request, and a corresponding bug issue has been created to track and resolve it.